spec/controllers/web/dashboard/users_controller_spec.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Web::Dashboard::UsersController, type: :controller do
+  let(:user) { create(:user, :staff) }
+
+  describe 'GET #index' do
+    context 'not signed in' do
+      before { get :index }
+
+      it 'redirect to root landing' do
+        expect(response).to redirect_to '/'
+      end
+    end
+
+    context 'not authorized' do
+      let(:user) { create(:user, :active) }
+      before { get :index }
+
+      it 'redirect to dashboard root' do
+        expect(response).to redirect_to dashboard_root_path
```

Comment:
An unauthorized user should be redirected to `root_landing_page`.
He is not able to access dashboard.
Please check this behaviour at local environment.